cmd/cdi/cmd/monitor.go

```diff
@@ -81,49 +81,41 @@ func monitorSpecDirs(args ...string) {
 	done = make(chan error, 1)
 
 	go func() {
+		// don't print the more often than this
+		lowPassFilter := 1 * time.Second
```

Comment:
Nit: I think calling this `oneSecond` instead will make the code below more readable.

Reply:
Updated.